s3response/s3response.go

```diff
@@ -384,3 +384,9 @@ type ListVersionsResult struct {
 	VersionIdMarker     *string
 	Versions            []types.ObjectVersion `xml:"Version"`
 }
+
+type GetBucketVersioningOutput struct {
+	XMLName   xml.Name `xml:"http://s3.amazonaws.com/doc/2006-03-01/ VersioningConfiguration" json:"-"`
+	MFADelete *types.MFADeleteStatus
+	Status    *types.BucketVersioningStatus
```

Comment:
I think these need an omitempty xml tag

Reply:
oh, does nil achieve this already? I didnt realize these are pointers before. that might make it so oomitempty is not needed.